pkg/apis/internal.admin.acorn.io/v1/computeclass.go

```diff
@@ -56,12 +56,12 @@ func ParseComputeClassMemory(memory ComputeClassMemory) (memoryQuantities, error
 	quantities.Def = &defInt
 
 	quantities.Values = make([]*resource.Quantity, len(memory.Values))
```

Comment:
This caused the array to have two nil values which was read in `memoryInValues()` function and not nil checked. Fixed both issues.